src/cli.rs

```diff
@@ -0,0 +1,135 @@
+mod info;
+mod init;
+
+use std::env;
+use std::io;
+use std::path::PathBuf;
+use std::process;
+
+use clap::arg_enum;
```

Comment:
This is the only direct use of `clap`. I didn't find a way to do the same with `structopt`, but since `structopt`  depends on `clap` it's not a big deal.